usr/lib/qubes-whonix/replace-ips

```diff
@@ -208,13 +226,29 @@ def main(args):
                 print("ERROR: 'qubesdb-read /qubes-ip' or 'qubesdb-read /qubes-gateway' failed!")
                 return
 
+            try:
+                ## FIXME: 'qubesdb-read /qubes-ip6' could fail if NetVM is set to 'none'.
+                current_ip6_local = subprocess.check_output(['qubesdb-read', '/qubes-ip6']).decode().rstrip()
+                ## FIXME: 'qubesdb-read /qubes-gateway6' could fail if NetVM is set to 'none'.
```

Comment:
This could (and probably will) also fail if the "ipv6" feature is not set on the appropriate qube(s), thus this needs to be fixed before Whonix 18's release.

Reply:
Created a fix on my end.